internal/constants/scripts.go

```diff
@@ -128,6 +148,11 @@ function update_physical_nics_references() {
         for nmfile in $host_work_dir/*.nmconnection ; do
           sed -i -e "s/=$logical_nic$/=$host_nic/g" -e "s/=$logical_nic\./=$host_nic\./g" $nmfile
           echo "Info: Using logical interface name '$logical_nic' for interface with Mac address '$mac_address', updated $nmfile"
+
+          for vlan in "${!vlan_replacements[@]}"; do
+            sed -i -e "s/=${vlan}$/=${vlan_replacements[${vlan}]}/g" -e "s/=${vlan}/=${vlan_replacements[${vlan}]}/g" $nmfile
```

Comment:
Because the key in `vlan_replacements` `logical_nic.<vlan>` which will never be in the `logical_mac_nic_map`

This key is what we need to replace to conform with the character limit.
In this case we'd be replacing something like `enp94s0f0np0.2507` with `enp94s0f0n.2507`

Reply:
I asked why the double pattern to sed.

Reply:
Ah I copied from the previous one.
Looks like that's been in there since https://github.com/openshift/assisted-service/pull/1274, but you're right. I can remove it from both if you want.

Reply:
Ah no, there's a literal dot in the previous pattern, but not in mine, which makes sense.

I'll just fix mine